pkg/collector/api/api.pb.go

```diff
@@ -1,7 +1,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.28.0
-// 	protoc        v3.21.12
+// 	protoc-gen-go v1.28.1
+// 	protoc        v3.21.9
```

Comment:
Not sure why this downgraded here.

Reply:
@mrueg I set up the CI installer to use 21.21, if this is regenerating locally with a lower version then your local install might be behind? As is this will fail the vet CI job.

https://github.com/kubernetes-sigs/usage-metrics-collector/blob/519fe97b531ff1e0f6866e638d913d183813c3b5/tools/install-protoc.sh#L7